build.gradle

```diff
@@ -0,0 +1,65 @@
+plugins {
+    id 'java'
+}
+
+repositories {
+    jcenter() // You can use any other repository you need
+}
+
+dependencies {
+    testImplementation 'junit:junit:4.12'
+    // Add more dependencies as needed
+}
+
+// Define source and test source directories
+sourceSets {
```

Comment:
We are not changing or adding any other folder. Going with the standard Java folder structure.
Do we need this?

Reply:
removed sourceSet.